docs/markdown/Syntax.md

```diff
@@ -220,6 +220,14 @@ is_x86 = target.startswith('x86') # boolean value 'true'
 is_bsd = target.to_lower().endswith('bsd') # boolean value 'true'
 ```
 
+#### .substring()
+
+```meson
+target = 'x86_FreeBSD'
+platform = target.substring(0, 3) # prefix string value 'x86'
+system = target.substring(4) # suffix string value 'FreeBSD'
+```
```

Comment:
Somewhat unfortunate example, because here `split('_')` would arguably be a better fit, but doesn't matter I guess :)

Reply:
That's why I implemented this because gstdockingstation.split('gst') is giving you ockination :)

Reply:
I agree but as a new manner to do it and be sure of of the result :)